checker-qual/src/main/java/org/checkerframework/checker/lock/qual/GuardedByBottom.java

```diff
@@ -22,6 +22,14 @@
 @Documented
 @Retention(RetentionPolicy.RUNTIME)
 @Target({ElementType.TYPE_USE, ElementType.TYPE_PARAMETER})
-@TargetLocations({TypeUseLocation.EXPLICIT_LOWER_BOUND, TypeUseLocation.EXPLICIT_UPPER_BOUND})
+@TargetLocations({
+    TypeUseLocation.LOWER_BOUND,
+    TypeUseLocation.UPPER_BOUND,
+    TypeUseLocation.PARAMETER,
```

Comment:
This is because these qualifiers are used on parameters in some test files. For example, this bottom qualifier is used in `checker/tests/lock/Strings.java:14`